### PR TITLE
fix(escalating): Add more docs links to archive button

### LINKED ISSUE #51956: Add additional links to docs for archive/escalation education
AC:
- Dropdown for Archive button on issue details and stream should have a link next to the `Until Escalating` option.  The link should go to https://sentry-docs-git-update-beta-test-archiving.sentry.dev/product/issues/states-triage/escalating-issues/
- Hover over the Archive button on the issue stream and details page should include a link to docs.  https://sentry-docs-git-update-beta-test-archiving.sentry.dev/product/issues/states-triage/

### PR DESCRIPTION
**Pull request by scttcper**
- adds a link in the tooltip on button hover
- adds link in the dropdown

image
image

fixes https://github.com/getsentry/sentry/issues/51956

### CODE REVIEW

File: static/app/components/actions/archive.tsx [hunk 1]
```diff
@@ -136,7 +142,19 @@ function ArchiveActions({
...
           />
         )}
-        menuTitle={t('Archive')}
+        menuTitle={
+          <MenuWrapper>
+            {t('Archive')}
+            <StyledExternalLink
+              href="https://sentry-docs-git-update-beta-test-archiving.sentry.dev/product/issues/states-triage/escalating-issues/"
+              onClick={event => {
+                event.stopPropagation();
```
Comment (evanpurkhiser) [inline static/app/components/actions/archive.tsx:151]: What's this needed for?

File: static/app/components/actions/archive.tsx [hunk 2]
```diff
@@ -136,7 +142,19 @@ function ArchiveActions({
             disabled={disabled}
           />
         )}
-        menuTitle={t('Archive')}
+        menuTitle={
+          <MenuWrapper>
```
Comment (evanpurkhiser) [inline static/app/components/actions/archive.tsx:146]: @vuluongj20 This is the best way to do this yeah?

**Review by evanpurkhiser — APPROVED** [review]

File: static/app/components/actions/archive.tsx (hunk 1 above)
Comment (scttcper) [reply static/app/components/actions/archive.tsx:151]: whoops thanks i was playing with something else

File: static/app/components/actions/archive.tsx (hunk 2 above)
Comment (vuluongj20) [reply static/app/components/actions/archive.tsx:146]: Yeah…

**Pull request merged**